htdocs/accountancy/class/bookkeeping.class.php

```diff
@@ -2707,7 +2715,7 @@ public function closeFiscalPeriod($fiscal_period_id, $new_fiscal_period_id, $sep
 				$sql .= ' AND aa.pcg_type IN (' . $this->db->sanitize(implode(',', $pcg_type_filter), 1) . ')';
 				$sql .= " AND DATE(t.doc_date) >= '" . $this->db->idate($fiscal_period->date_start) . "'";
 				$sql .= " AND DATE(t.doc_date) <= '" . $this->db->idate($fiscal_period->date_end) . "'";
-				$sql .= ' GROUP BY t.numero_compte, t.label_compte, aa.pcg_type';
+				$sql .= ' GROUP BY t.numero_compte, aa.pcg_type';
```

Comment:
Imagine you have in database 
1 line:   account = 12345,  label = 'A label for the transaction", amount  10
1 line:   account = 12345, label = 'A different transaction',  amount 15

What do we want into the A nouveau ?
Just 1 line with   amount 25 and label = 'A nouveau...'   ?

Can you confirm ?

Reply:
I can confirm that this is exactly what we're aiming for. It's the a-nouveaux principle.

In France, this consists of summarizing all class 1 to 5 accounts (CAPIT to FINAN) in a single account.

And the annual result class 7 - 6 (INCOME - EXPENSE) goes to account 120 or 129.

Reply:
So PR seems good to me.